site/src/components/AuditLogRow/AuditLogRow.tsx

```diff
@@ -81,51 +85,61 @@ export const AuditLogRow: React.FC<AuditLogRowProps> = ({
           <Stack
             direction="row"
             alignItems="center"
-            justifyContent="space-between"
-            className={styles.auditLogRowInfo}
+            className={styles.auditLogHeaderInfo}
           >
-            <Stack direction="row" alignItems="center">
+            <Stack
+              direction="row"
+              alignItems="center"
+              className={styles.fullWidth}
+            >
               <UserAvatar
                 username={auditLog.user?.username ?? ""}
                 avatarURL={auditLog.user?.avatar_url}
               />
-              <div>
-                <span
-                  className={styles.auditLogResume}
-                  dangerouslySetInnerHTML={{
-                    __html: readableActionMessage(auditLog),
-                  }}
-                />
-                <span className={styles.auditLogTime}>
-                  {createDayString(auditLog.time)}
-                </span>
-              </div>
-            </Stack>
 
-            <Stack
-              direction="column"
-              alignItems="flex-end"
-              spacing={1}
-              className={styles.auditLogRight}
-            >
-              <Pill
-                type={pillTypeByHttpStatus(auditLog.status_code)}
-                text={auditLog.status_code.toString()}
-              />
               <Stack
+                alignItems="baseline"
+                className={styles.fullWidth}
+                justifyContent="space-between"
                 direction="row"
-                alignItems="center"
-                className={styles.auditLogExtraInfo}
               >
-                <div>
-                  <strong>IP</strong> {auditLog.ip ?? notAvailableLabel}
-                </div>
-                <div>
-                  <strong>OS</strong> {os.name ?? notAvailableLabel}
-                </div>
-                <div>
-                  <strong>Browser</strong> {displayBrowserInfo}
-                </div>
+                <Stack
+                  className={styles.auditLogResume}
```

Comment:
Wondering if "resume" is a false cognate here - is it intended to mean "summary"?

Reply:
Yes, it is. Thx